src/topology/algebra/ordered.lean

```diff
@@ -294,17 +294,14 @@ lemma is_preconnected.Icc_subset {s : set α} (hs : is_preconnected s)
   Icc a b ⊆ s :=
 by simpa only [image_id] using hs.intermediate_value ha hb continuous_on_id
 
-/-- The only unbounded preconnected subset of a conditionally complete lattice is the whole
-space. -/
+/-- The only unbounded preconnected subset of a linear order is the whole space. -/
```

Comment:
This docstring is still a little bit ambiguous, as the whole space is not always preconnected.

Reply:
Already updated:
```
/-- If preconnected set in a linear order space is unbounded below and above, then it is the whole
space. -/
```

Reply:
Ah yes, sorry.